guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Creating_an_Activation_Key_{context}"]
-= Creating an activation key
+= Creating an Activation Key
```

Comment:
We are opting for sentence case instead of title case for titles and headings. https://community.theforeman.org/t/sentence-case-for-foreman-guide-titles-and-headings/36565/1

We can start applying here as well.